Message.java

```diff
@@ -0,0 +1,20 @@
+
+public class Message {
+	
+	private String toAddress;
+	private String fromAddress;
+	private String message;
+	
+	public void getTo(String toAddress) {
```

Comment:
I always get the get/set confused. lol. Yeah, just kinda throwing some stuff together for now. Refine as we go, I figure.